3_visualize/src/visualize.R

```diff
@@ -0,0 +1,80 @@
+# Define visualize() function
+
+visualize <- function(repo_path){
+  # Set up environment
+  setwd(repo_path)
+  library(dplyr)
+  library(whisker)
```

Comment:
See the "overarching comments" at the start of the review. I would like you to pull this out into an R script that sits at the top of the file directory.